solana-programs/anchor/audius-data/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
We should try to re-use our existing configs as much as possible, which is a standard + prettier + eslint config. I think this misses some of those things (.e.g " should be ' in a lot of spots).

https://github.com/AudiusProject/stems/blob/74da5237e13320525a769e375f36d48f0300e617/.eslintrc.js#L12

Reply:
Obviously not all of the react stuff is pertinent

Reply:
@sliptype  maybe in the future, we can have an eslint config w/o the react stuff and inherit that + combine for the react repos

Reply:
Yup, I started that a while back but haven't had a chance to finish it. Will revisit soon

Reply:
@isaacsolo  this is probs fine for now then. gets us 90% of the way

Reply:
thanks, added standard and more prettier configs which caught some bugs :)